magpie/api/management/user/user_utils.py

```diff
@@ -98,14 +98,14 @@ def create_user_resource_permission(permission_name, resource, user, db_session)
                          detail=s.UserResourcePermissions_POST_CreatedResponseSchema.description)
 
 
-def delete_user_resource_permission(permission_name, resource, user_id, db_session):
+def delete_user_resource_permission(permission_name, resource, user, db_session):
```

Comment:
just add the typing as for `create_user_resource_permission`

`# type: (Str, models.Resource, models.User, Session) -> HTTPException`